.github/workflows/ci.yml

```diff
@@ -28,6 +28,13 @@ jobs:
     timeout-minutes: 60
 
     steps:
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
+        with:
+          detached: true
+          limit-access-to-actor: true
+        timeout-minutes: 15
+
```

Comment:
```suggestion
```
Turns out this isn't safe to merge to master (it's fine to use it in branches though)